aws/requests.yaml

```diff
@@ -2,7 +2,7 @@ releases:
     - name: ">= 15.0.0"
       requests:
         - name: app-operator
-          version: ">= 4.1.0"
+          version: ">= 4.2.0"
```

Comment:
After we had made performance fix, https://github.com/giantswarm/giantswarm/issues/16406, we released v4.2.0.